blaze-core/src/test/scala/org/http4s/blaze/TestHead.scala

```diff
@@ -35,6 +35,7 @@ abstract class TestHead(val name: String) extends HeadStage[ByteBuffer] {
     closed = true
     super.stageShutdown()
     p.trySuccess(ByteBuffer.wrap(getBytes()))
+    ()
```

Comment:
-Ywarn-value-discard

Of the standard flags that all the cool kids use, it's the one that I'm the most skeptical of.  Especially now that procedure syntax is deprecated, and we don't get bitten by missing equals signs on functions with inferred return types.